docs/toucan-form/native-validation/index.md

```diff
@@ -34,3 +34,26 @@ const data: MyFormData = {};
   <form.Input @label='Last name' @name='lastName' />
 </ToucanForm>
 ```
+
+## Submit and Reset Actions
+
+The component yields back `submit` and `reset` actions for more complex cases of submitting or resetting your form data.
+
+**NOTE:** Calling `submit` directly is **not** required for most cases. The implementation only requires a button tag with the `type="submit"` attribute set.
```

Comment:
✅  Thanks for adding this, and also pointing out that's it's not needed most of the time.